R/checkInputList.R

```diff
@@ -1,4 +1,8 @@
 checkInputList <- function(inputList, mistypes = c("mnr", "mbd", "mir", "mbi")) {
+  checkmate::assert_list(inputList, types = "data.frame")
+  checkmate::assert_character(mistypes)
+  #checkmate::assert_subset(mistypes, choices = c("mnr", "mbd", "mir", "mbi"))
```

Comment:
I wasn't sure which assertion I should use here

Reply:
could be this ("mnr", "mbd", "mir", "mbi") and much more but could also be numeric like -99 etc. so please check character or numeric